news/2021-11-08-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,132 @@
+
+The difficulty that Flashlight adds to a map is now measured much more accurately, thanks to [**MBmasher**](https://osu.ppy.sh/users/4498616)'s change in [this pull request](https://github.com/ppy/osu/pull/14217).
+
+Previously, the object count-based method posed a number of issues. Flow aim (used for hitting streams) is mostly represented by the speed skill, not aim. Additionally, the aim skill, unsurprisingly, only measures the aim difficulty, not the Flashlight difficulty. To put this simply, there will be types of jumps that are hard to aim but quite easy to read with Flashlight, and vice versa.
+
+For example, let's consider a familiar type of jump pattern: a back and forth. This is an example of a jump that can have high aim difficulty, but relatively low Flashlight difficulty. The distance between the first object and the third object in a back and forth is typically small enough for it to be within the Flashlight radius, thus allowing a player to read the position of that circle very easily.
+
+![](/wiki/shared/news/2021-11-08-performance-points-star-rating-updates/visible-fl-area.png)
+
+The creation of a Flashlight skill makes it possible to have it contribute to star rating. [This has been enabled](https://github.com/ppy/osu/pull/14753) following the results of the recent osu! performance points and star rating survey, which revealed that players desire star rating to be a metric of difficulty, and that they want subjective skills included. Ask and thou shall receive!
+
+It is also worth noting that the Hidden mod now applies a 1.3x multiplier on the Flashlight skill. The visibility of hit objects are decreased significantly due to the fading of circles and the absence of approach circles.
+
+Find yourself curious about the details? [MBmasher](https://osu.ppy.sh/users/4498616)'s write-up about the implementation will very likely scratch that itch, so feel free to [**indulge in some Flashlight theory**](https://docs.google.com/document/d/11FBEEPXK8yI2d5uTZuebJC3S9nVUKgMD8WdvblERyYA/edit).
+
+### Speed hard cap removal
+
+The speed hard cap in osu! has been removed by [**emu1337**](https://osu.ppy.sh/users/2185987) and [**Apo11o**](https://osu.ppy.sh/users/9558549) in [this pull request](https://github.com/ppy/osu/pull/14617), allowing 300+ BPM streams to be weighed accurately. 
+
+The speed cap was previously set to 300 BPM to prevent abuse cases, which means that calculating difficulty on beatmaps with higher BPM requires special consideration. Two adjustments were made to account for that:
+
+- Speed is nerfed on doubles that are double-tappable. If the time since the last object for the current object is smaller than the hit window for a 300, we instead take the difference between that object and the hit window of the last one.
+- Speed is nerfed if successive objects are faster than the 300 hit window. This is intended to target cheesing of faster bursts and streams if the hit window is way larger than the time between notes.
```

Comment:
Would like this addressed. Main concern I guess is that "speed is nerfed" doesn't make much sense as a statement. "Speed pp" would also be less to-the-point. Can be a separate point still, but it could use some less redundant wording.

Reply:
fair point, was holding off on this since I'm not 100% sure on either wording, but we can go with yours for the time being